scripts/storybook/utils.js

```diff
@@ -162,7 +162,6 @@ function getPackageStoriesGlob(options) {
       const storiesGlob = '**/@(index.stories.@(ts|tsx)|*.stories.mdx)';
       const pkgMetadata = getProjectMetadata({ name: pkgName });
 
-      //TODO: simplify once v9 migration [https://github.com/microsoft/fluentui/issues/24129] is complete.
```

Comment:
Starting storybook will error out since certain v9 packages don't have stories and thus don't have a `stories/` subfolder. This means that this will now require a default fallback if a stories subfolder doesn't exist so will be keeping the conditions and removing the TODO comment

Reply:
sounds good, ty